add.test.js

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, test } from '@jest/globals';
+import Activities from './src/activities.js';
+// eslint-disable-next-line no-unused-vars
+import init from './localstorageMock.js';
```

Comment:
Please don't disable the linter, I recommend that you allow it to run in order to ensure you are following best practices. Furthermore, I suggest making use of the localStorage mock in your test in order to meet all of the project requirements.

NOTE: I recommend doing the same for the removed test and as the requirement stated, please merge both tests into one file.